cmd/acra-translator/server/server.go

```diff
@@ -41,27 +42,27 @@ import (
 // ReaderServer represents AcraTranslator server, connects with KeyStorage, configuration file,
 // gRPC and HTTP request parsers.
 type ReaderServer struct {
-	config            *common.AcraTranslatorConfig
-	keystorage        keystore.TranslationKeyStore
-	connectionManager *network.ConnectionManager
-	grpcServer        *grpc.Server
-
-	httpDecryptor *http_api.HTTPConnectionsDecryptor
-
-	waitTimeout time.Duration
-
+	config                 *common.AcraTranslatorConfig
+	keystorage             keystore.TranslationKeyStore
+	connectionManager      *network.ConnectionManager
+	grpcServer             *grpc.Server
+	httpDecryptor          *http_api.HTTPConnectionsDecryptor
+	waitTimeout            time.Duration
 	listenersContextCancel []context.CancelFunc
 	grpcServerFactory      common.GRPCServerFactory
+	waitGroup              sync.WaitGroup
 }
 
 // NewReaderServer creates Reader server with provided params.
 func NewReaderServer(config *common.AcraTranslatorConfig, keystorage keystore.TranslationKeyStore, grpcServerFactory common.GRPCServerFactory, waitTimeout time.Duration) (server *ReaderServer, err error) {
+	var wg sync.WaitGroup
 	return &ReaderServer{
 		grpcServerFactory: grpcServerFactory,
 		waitTimeout:       waitTimeout,
 		config:            config,
 		keystorage:        keystorage,
 		connectionManager: network.NewConnectionManager(),
+		waitGroup:         wg,
```

Comment:
actually if we declare field as value, not as pointer, than we don't need to initialize it. golang initialize all values by their default values.. we can skip `var wg sync.WaitGroup`.

Reply:
Done in https://github.com/cossacklabs/acra/pull/406/commits/55e177ef6709e55c8d819195c9989dda72a96d1f